app/src/main/java/com/asfoundation/wallet/referrals/InviteFriendsVerificationFragment.kt

```diff
@@ -65,6 +66,11 @@ class InviteFriendsVerificationFragment : DaggerFragment(), InviteFriendsVerific
     super.onDestroyView()
   }
 
+  private fun convertToString(value: BigDecimal): String {
```

Comment:
This piece of code is duplicated in a few files.
Maybe consider creating an extention for BigDecimals